oaknational/Oak-Web-Application
# feat(favicon): update favicon to black bg + white icon

Update favicons to black bg + white icon to match with other social icons following death of QEII.

## Description

All favicons updated to black bg, white icon

## Issue(s)

Fixes #

## How to test

1. Go to localhost
2. Click on site
3. You should see black favicon

## Screenshots

How it used to look (delete if n/a):

<img width="1463" alt="Screenshot 2022-09-08 at 23 34 10" src="https://user-images.githubusercontent.com/4476971/189237727-3895b093-335c-4878-8f22-6a5809dd0499.png">

How it should now look:

<img width="1820" alt="Screenshot 2022-09-08 at 23 34 30" src="https://user-images.githubusercontent.com/4476971/189237738-6b7c072c-b40c-4267-b9a6-c659a3eb96d5.png">


## Checklist
